Dockerfile

```diff
@@ -7,18 +7,20 @@ COPY ./pororift-client-server.go .
 RUN go mod init pororift-client
 RUN go build ./pororift-client-server.go
 
-FROM ubuntu:19.04
-WORKDIR /app
-
-COPY --from=builder /go-source/pororift-client-server .
-RUN apt-get update && \
-        apt-get -y install sudo
-RUN apt-get install -y nodejs npm 
+FROM node:10 AS builder-node
+WORKDIR /node-source
 
 COPY . .
 
-RUN npm install && \
-        npm run build
+RUN npm cache verify
+RUN npm install 
+RUN npm run build
+
+FROM centos:7
+WORKDIR /app
+
+COPY --from=builder-node /node-source/dist ./dist/
+COPY --from=builder-go /go-source/pororift-client-server .
```

Comment:
Just curious, what is this go for?

Reply:
so one image to build the node stuff and another to build the go stuff that way we can pull pre-built images instead of installing a bunch of stuff like it was before.

Reply:
was it for the echo server?

Reply:
yea the one that was serving the client basically